test/functional/system/transactions/3.3.votes.js

```diff
@@ -191,11 +191,7 @@ describe('system test (type 3) - voting with duplicate submissions', () => {
 
 					it('adding to pool downvoting transaction to same delegate from same account should fail', done => {
 						localCommon.addTransaction(library, transaction4, err => {
-							expect(err).to.equal(
-								`Failed to remove vote, delegate "${
-									accountFixtures.existingDelegate.delegateName
-								}" was not voted for`
-							);
+							expect(err).to.include('Transaction is already confirmed');
```

Comment:
Test scenario
`adding to pool downvoting transaction to same delegate from same account should fail`
is totally contradict to expectation. While the pervious exception was exact match to test statement.